ui/StatusQ/src/StatusQ/Components/StatusChatList.qml

```diff
@@ -185,9 +190,26 @@ Item {
                         notificationsCount: model.notificationsCount
                         highlightWhenCreated: !!model.highlight
                         selected: (model.active && root.highlightItem)
+
+                        asset.isImage: !!model.icon
                         asset.emoji: !!model.emoji ? model.emoji : ""
-                        asset.color: !!model.color ? model.color : Theme.palette.userCustomizationColors[model.colorId]
-                        asset.name: model.icon
+                        asset.color: isContactIcon ? Theme.palette.indirectColor2 : (!!model.color ? model.color : Theme.palette.userCustomizationColors[model.colorId])
+                        asset.bgColor: isContactIcon ? Theme.palette.userCustomizationColors[model.colorId] : "transparent"
+                        asset.name: {
+                            if (asset.isImage) {
+                                return model.icon
+                            }
+                            if (isContactIcon) {
+                                return "contact"
+                            }
+                            return ""
+                        }
+                        asset.width: isContactIcon ? iconWidth * 0.9 : iconWidth
```

Comment:
Why is this scaling the icon down? :thinking:

Reply:
The icon was too big when I was working on it and I thought this is what fixed it, but turns out it's not, I just tried removing it and it doesn't change anything.

I think I need to find a way to fix it though, because it's still too big compared to the design.

Design:
![image](https://github.com/user-attachments/assets/33b70172-b2cd-496d-a0e1-2606d06ee8f6)

Implementation:
![image](https://github.com/user-attachments/assets/d118d747-76fc-4261-8c7c-06e368242253)